### PR TITLE
Translation of JSX components, Infite scroll skeleton

### PR DESCRIPTION
**Pull request by lukybil**
- _requires new build, commit number: 865e1dce3ecb13f91d1f4c8749956e1a1617e5d8_
- added links support for Markdown #18
- added links to translations for some app texts
- added skeletons for infinite lists #181

### CODE REVIEW

File: components/screen-layout/ContentWithAvatar.tsx [hunk 1]
```diff
@@ -11,6 +12,7 @@ type ContentWithAvatarProps = {
   variant?: 'success' | 'error'
   customAvatarComponent?: ReactNode
   actionButton?: ReactNode
+  markdown?: boolean
```
Comment (radoslavzeman) [inline components/screen-layout/ContentWithAvatar.tsx:15]: We try to indicate boolean props by proper name, usually using `is` prefix, or verb prefix. It also make sense to use `asSomething` (e.g.`asChild` in `Link` component).
From our Strapi best practices docs: https://bratislava.github.io/strapi/best-practices#field-naming

I would say, `asMarkdown` is okay for this case?

File: package.json [hunk 2]
```diff
@@ -59,13 +60,15 @@
...
     "react-native-gesture-handler": "~2.12.0",
+    "react-native-linear-gradient": "^2.8.3",
     "react-native-markdown-display": "~7.0.0-alpha.2",
     "react-native-mmkv": "~2.10.2",
     "react-native-pager-view": "6.2.0",
     "react-native-reanimated": "~3.3.0",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.0",
     "react-native-shadow-2": "~7.0.8",
+    "react-native-skeleton-placeholder": "^5.2.4",
```
Comment (radoslavzeman) [inline package.json:71]: Please use `~` versions of newly installed libraries.

**Review by radoslavzeman — APPROVED** [review]
Please address comments :)

File: components/screen-layout/ContentWithAvatar.tsx (hunk 1 above)
Comment (lukybil) [reply components/screen-layout/ContentWithAvatar.tsx:15]: a good point 👍 renamed to `asMarkdown`

**Pull request merged**